src/main/features/dashboard/views/macro/claimStatus/claimPartAdmitted.njk

```diff
@@ -0,0 +1,90 @@
+{% from "externalLink.njk" import externalLink %}
+
+{% macro claimPartAdmittedAlreadyPaidForDefendantDashboard(claim) %}
+  {{ t('We’ve emailed {{ defendantName }} telling them when and how you said you paid the claim.',
```

Comment:
Does not look right, it should be `{{ claimantName }}` because macro is for defendant dashboard.